.travis.yml

```diff
@@ -1,5 +1,5 @@
 sudo: required 
-dist: trusty 
+dist: xenial
```

Comment:
Just FYI, Travis CI supports up to Ubuntu Trusty as of today to my knowledge, but it looks like using unavailable distro name doesn't harm (Travis CI [just falls back to Trusty without any message](https://travis-ci.org/ros-industrial-consortium/packml/jobs/218135753#L17) etc.) so we can leave this as it is.